control-plane/connect-inject/endpoints_controller_test.go

```diff
@@ -5685,6 +5864,98 @@ func Test_GetWANData(t *testing.T) {
 	}
 }
 
+func TestCreateTerminatingGatewayRegistrations(t *testing.T) {
+	cases := map[string]struct {
+		podName          string
+		serviceName      string
+		serviceNamespace string
```

Comment:
It looks like these fields are constant across every test. id just move them into the fixture.

Reply:
Cool, I originally parameterized this with the thought that you would have similar tests for mesh gateways that I would combine with this set.